src/State/AuthFlow/reducers.js

```diff
@@ -0,0 +1,29 @@
+import initialState from '../../Reducers/initialState';
+import * as types from './types';
+
+const authReducer = (state = initialState.currentUser, action) => {
+  switch (action.type) {
+    case types.AUTHENTICATE_FETCH: {
+      return { ...state };
+    }
+    case types.AUTHENTICATE_SUCCESS: {
+      return {
+        ...state,
+        AccessKeyId: action.payload.AccessKeyId,
+        EmpEmail: action.payload.EmpEmail,
+        SamlHash: action.payload.SamlHash,
+        SecretKey: action.payload.SecretKey,
+        SessionToken: action.payload.SessionToken,
+        UserKey: action.payload.UserKey
+      };
+    }
+    case types.AUTHENTICATE_FAILURE: {
```

Comment:
maybe you might want to add `error` state here. If users get fail to aut

Reply:
I want to handle that separately from this MR, but that would be the idea